### DISCUSSION TITLE
Easier way to select which building we want to build

### DISCUSSION THREAD

**Discussion by mathieumahe** [Keriew/augustus#616] [Ideas]
Hello,

Is it possible to add a new feature : select which building I want to build by shift + clicking upon an already existing building of the same type ?
For example, if I want to add new pottery workshops, I can shift + click on an existing pottery workshop to select it, and then click as usual to place them.

We can also imagine (in the same spirit of discussion #342 ) shift+click on ruins to select what it once was, and then being able to construct above ruins, replacing them with the selected building.

Thanks for the great work !

**Comment by Suprcheese** [comment]
There is already a bindable hotkey for this function, to select the building under the cursor. Check the hotkeys settings in game. I have mine bound to Q.

**Comment by mathieumahe** [comment]
Oh that's awesome, thanks!

So, maybe it is possible to add the same feature for destroyed buildings and then allowed to rebuild directly above the ruins ? It would really accelerate the reconstruction of a destroyed neighbourhood.

**Reply by crudelios** [reply]
This has been a feature that's been asked quite a lot actually! Eventually I'll have a look at this.

**Reply by mathieumahe** [reply]
I think, enabling cloning to a destroyed building is not that hard, so I can check that during the week-end if you want.

I guess, replacing ruin by a new building is probably harder.
